cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/LoadMonitor.java

```diff
@@ -500,6 +545,32 @@ public ClusterModel clusterModel(long from,
     return clusterModel;
   }
 
+  /**
+   * Populate replica placement information, i.e. replica resides on which disk of the broker, to the cluster model.
+   *
+   * @param clusterModel The cluster model to populate replica placement information.
+   */
+  private void populateReplicaPlacementInfoForClusterModel(ClusterModel clusterModel) {
```

Comment:
As discussed above, this logic should be as part of `populateLoad` (if requested so).